utils/appliance/__init__.py

```diff
@@ -124,6 +124,10 @@ def __init__(self, address=None, browser_steal=False, container=None):
             [self.browser])
         self._server = None
 
+    def get(self, obj, *args, **kwargs):
```

Comment:
that sould be `cls`, not `obj` ?

to make mistakes more visible i propose always passing appliance as python kwarg, that way it automatically fails on dict unpacking

for better errors we could throw in a assertion